content/backend/graphql-js/1-getting-started.md

```diff
@@ -219,6 +219,8 @@ In this case, we have three root fields: `users` and `user` on `Query` as well a
 
 What are the API operations that can be derived from this schema definition? Well, we know that each API operation always needs to start with a root field. However, we haven't learned yet what it looks like when the _type_ of a root field is itself another [object type](http://graphql.org/learn/schema/#object-types-and-fields). This is the case here, where the types of the root fields are `[User!]!`, `User` and `User!`. In the `info` example from before, the type of the root field was a `String`, which is a [scalar type](http://graphql.org/learn/schema/#scalar-types).
 
+// TODO (robin-macpherson): question for Nikolas
+
 - `typeDefs`: These are the type definitions from your application schema.
 - `resolvers`: This is a JavaScript object that mirrors the `Query`, `Mutation` and `Subscription` types and their fields from your application schema. Each field in the application schema is represented by a function with the same name in that object.
 - `context`: This is an object that gets passed through the resolver chain and every resolver can read from or write to.
```

Comment:
@nikolasburk as I was reading back through the original section here, I feel like there is something missing here? We talk about Scalar Types and Object Types and then 💥boom, 3 bullet points explaining `typeDefs`, `resolvers`, and `context`. It feels quite abrupt. Feels like there should be a sentence right before the three bullet points to lead into it.

Reply:
Hmm I agree, this seems like an error actually... might have been a leftover from a previous iteration 🙈  I don't think the bullets are properly placed in this section at all, just removing them I think would probably resolve this?

Reply:
Yep it seemed like an error to me, too. I was going to just try to fix it but I simply couldn't see how the bullets made sense here 😅 I agree that simply removing them resolves is since the following paragraph leads on perfectly 😄 Done ✅